test/robustness/within/Jamfile

```diff
@@ -0,0 +1,17 @@
+# Boost.Geometry (aka GGL, Generic Geometry Library)
+# Robustness Test - convex_hull
```

Comment:
this is not convex hull, typo?

Reply:
Indeed, I copied another file and forgot to adapt the title.
:heavy_check_mark: fixed

Reply:
> Thanks, this is cool! I am OK with merging now. I will have a deeper look next week. Regarding the name I also agree it is neither quad nor kd-tree (it is not a tree at all of course and it actually combines the default splitting strategy "cut in middle" of quad trees with the kd-tree per-dimension partitioning). So I think it is better to just call it "partitioning".

Renamed now to `partitioning` (currently in namespace `experimental`)